amplify/functions/send-email/email-template.html

```diff
@@ -0,0 +1,229 @@
+          arQTPUL/0KLZ9JaoYDPpX0tQBXhfNxoTILZbgafuX7vN8Fr7r66QF38T+sA0yOlXli2xkbgbKd8d
+          PTYjbxHPqxDitgJvzxmiptNaAn4HliO0OTny6ZfFa8fPm37PuGDr5L+UarMP36uYyqhNZ4haDZKE
+          DhxUxy/ixufq6bfOmX/fyKZo3lsVeX9Qhw37tp0hajVwEDK4JBNirXfGINRqCLY9k82a5tGrWeX1
+          LITQJh9niKa7Ly2bxTnY985SslCLCjZPhwBr8TiI4HH+WuqXkMdrZZrHKpnSMX3w8/JE+ZR8BVvi
+          h8XjWQged/SQm/dM8jWivs4Q3WjRe5tIYTDx4oKLM1eDOymatuJgoxtYHtqUux4M85nYnavJGaJr
+          yY4Znu4Za9KFM3v4nKtDpIMqNqutYTx8MJqb32kGW5vO1bQdJAmZ5tHO/rXVQDtb/OalT1yFWnDF
+          ZrKjh5N5ZNKn5WX2eMzGhu3a0cPXJOS2DRxIoMmE26OKG0WaBFtTVxxs1E9QPOvjlxPVv8ZSqmTX
+          uuuVk2VzVAJXpnw0dRG8TN+QXW8XMjipKyzYWtC/VpGJmPKW9tC8iDrjwGjDxZgqUi3YHJ4hKrvE
+          yNcL7y6VS6ukGCg3kew+OzkHnbzoZS21/Jl6C/j6m6Lb29MUrX6ZHoIt5vQnqyrGw8CB9zNEJfiP
+          nrj6c1OtWkMq95SM5nruk5PncHWr8otZHyy+pXjy/Wv9/tKeBq846KXaVjn5TRa4B77lz5v61HfL
+          a5WdarWmtcgAiIRdtW33RIJ93CTElr+5Wr6srM5ccBVsAGysLhW7qQw9mUJSBt6vVwOv+vchL5+1
+          q2hk+ZuE9DBnKBBsAJDYCB8BAIINAAg2ACDYAIBgAwCCDQDBBgAEGwAQbABQn9HdE2OX+RgANMmW
+          a9dYUQWApigAEGwAQLABAMEGAAQbAIINAAg2ACDYAIBgAwCCDQDBBgAEGwAQbABAsAEAwQaAYAMA
+          gg0ACDYAINgAgGADAIINAMEGAAQbABBsAECwAQDBBoBgAwCCDQAINgAg2ACAYANAsAEAwQYABBsA
+          EGwAQLABINgAgGADAIINAAg2ACDYAIBgA0CwAQDBBgAEGwAQbABAsAEg2ACAYAMAgg0ACDYAINgA
+          EGwAQLABAMEGAAQbABBsAAg2ACDYAIBgAwCCDQAINgAg2AAQbABAsAEAwQYABBsAEGwACDYAINgA
+          gGADAIINAAg2AAQbABBsAECwAQDBBgAEGwCCDQAINgAg2ACAYAMAgg0ACDYABBsAEGwAQLABAMEG
+          AAQbAIINAAg2AHDhfwIMAJCNcOrSAj7MAAAAAElFTkSuQmCC" alt="NHS Logo" width="90" />
+              </th>
+              <td>
+                <p class="notifyHeading">Notify</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p class="infoMessage">You have
+                  successfully submitted your template to NHS Notify.</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p class="dataLabel">Template name</p>
+                <p class="dataValueShort">{{templateName}}</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p class="dataLabel">Template ID</p>
+                <p class="dataValueShort">{{templateId}}</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p class="dataLabel">Template content</p>
+                <p class="dataLabelDetail">
+                  This content does not have your formatting applied. This is not how your message will look when it's sent to
+                  recipients.
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2" class="dataValueTemplateContentWrapper">
+                <p class="dataValueTemplateContent">{{{templateMessage}}}</p>
+              </td>
+            </tr>
+          </table>
```

Comment:
Very minor thing. But the spacing between this end table tag and the end body tag don't seem to align. 

Probably because our linting isn't picking the html files? Does it matter?

Reply:
One for @alibombaye to look at in his next update.